packages/vite/src/node/plugins/html.ts

```diff
@@ -300,6 +300,9 @@ function handleParseError(
     case 'non-void-html-element-start-tag-with-trailing-solidus':
       // Allow self closing on non-void elements #10439
       return
+    case 'unexpected-question-mark-instead-of-tag-name':
+      // Allow <?xml> declaration and <?> empty elements
```

Comment:
```suggestion
      // Allow <?xml> declaration and <?> empty elements
      // lit generates <?>: https://github.com/lit/lit/issues/2470
```
Let's add why we are ignoring this potential user error.